ui/app/templates/partials/upgrade-overlay.hbs

```diff
@@ -62,14 +62,14 @@
             <li>Sentinel Integration</li>
             <li>Control Groups</li>
             <li>Seal Wrap / FIPS 140-2 Compliance</li>
-            <li>Gold support: 24x7 support w/SLA</li>
+            <li>Gold support: 24&times;7 support w/SLA</li>
           </ul>
         </div>
       </div>
     </div>
     <p class="has-text-centered">
-      <a href="https://www.hashicorp.com/go/vault-enterprise" class="button is-primary has-icon-right" target="_blank">
-        Request Info
+      <a href="http://hashicorp.com/products/vault/trial?source={{trackingSource}}" class="button is-primary has-icon-right" target="_blank">
```

Comment:
I think we should add `rel="noreferrer noopener"` here too.

Reply:
Oops and https://

Reply:
Good idea. Done